<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Persone</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Aggiungi</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cognome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
   </widget>
   <widget class="QLabel" name="TipoDipendente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo Dipendente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CognomeText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TipoText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Numb">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeText_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CognomeText_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TipoText_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Nome_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cognome_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
   </widget>
   <widget class="QLabel" name="TipoDipendente_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo Dipendente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Visualizza">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>16</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N°</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumeroMassimo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N° Persone</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
